### DISCUSSION TITLE
Array button images not displaying

### DISCUSSION THREAD

**Discussion by hechtcarmel** [rjsf-team/react-jsonschema-form#2904] [Q&A]
Hi,
For some reason, the buttons which should control the "add"/"remove" actions when using arrays are displayed incorrectly:
![image](https://user-images.githubusercontent.com/36325296/177030579-f5f36b9d-07f4-426a-ab98-2a0868fd5cbd.png)

I tried playing with the css, to no success.
Any help will be apprciated!

I am using the following schema:
`{
    title: "Image",
    type: "object",
    required: ["title"],
    properties: {
      named_location:{
            title: "Named Location",
            description: "A name for the camera location",
            type: "string"
        },
        speed: {
            title: "Speed",
            description: "Camera speed, user defined format",
            type: "string"
        },
        accelaration: {
            title: "Accelaration",
            description: "Camera accelaration, user defined format",
            type: "string"
        },
        arr: {
            title: "arr",
            description: "arr",
            type: "array",
            items: {
                type: "string"
              }
            },
            
        },
    
}`

**Comment by epicfaace** [comment]
Which theme are you using? Did you include the right CSS / setup required
for that theme?

--
Ashwin Ramaswami
